src/library/scala/math/package.scala

```diff
@@ -215,9 +218,37 @@ package object math {
     */
   def signum(x: Long): Long     = java.lang.Long.signum(x)
   /** @group signum */
-  def signum(x: Float): Float   = java.lang.Math.signum(x)
+  def signum(f: Float): Float   = java.lang.Math.signum(f)
   /** @group signum */
-  def signum(x: Double): Double = java.lang.Math.signum(x)
+  def signum(d: Double): Double = java.lang.Math.signum(d)
+
+  def floorDiv(x: Int, y: Int): Int = java.lang.Math.floorDiv(x, y)
```

Comment:
@golem131 - For `floorDiv` and 'floorMod` a group names _Modulus and Quotient_ fits in with the other group name and is clear.

While this group contains just these four methods a short note in `@groupdesc` could be added

> Calculate quotient values by rounding to negative infinity.